.editorconfig

```diff
@@ -8,3 +8,6 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 quote_type = single
+
+[cases.ts]
+trim_trailing_whitespace = false
```

Comment:
I needed this to stop webstorm removing all the trailing newlines in the `convertAnnotatedSourceToFailureCase` helper (can remove and file a separate PR though)